client/js/s3/request-signer.worker-manager.js

```diff
@@ -0,0 +1,88 @@
+/* globals qq, CryptoJS */
+
+/**
+ * Manages creation and communication of s3 signature workers.
+ */
+qq.s3.RequestSignerWorkerManager = function (o) {
+    "use strict";
+    var _worker = null,
+        _workerPromises = {},
```

Comment:
Minor detail, but that would be great. Elsewhere in the project, underscores are used to mark publicly-accessible methods/props that are only meant to be used internally (a poor-man's private).